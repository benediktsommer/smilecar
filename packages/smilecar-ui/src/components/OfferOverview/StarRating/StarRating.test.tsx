import { render } from '@testing-library/react';

import { StarRating } from './';

describe('Test <StarRating />', () => {
  it('should render an html markup', () => {
    const { asFragment } = render(
      <StarRating rating={{ average: 0.68, count: 3 }} />
    );
    expect(asFragment()).toMatchSnapshot();
  });
});
