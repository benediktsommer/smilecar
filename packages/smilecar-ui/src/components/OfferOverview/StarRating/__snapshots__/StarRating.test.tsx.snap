// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test <StarRating /> should render an html markup 1`] = `
<DocumentFragment>
  <div
    class="MuiBox-root MuiBox-root-1"
  >
    <div
      class="MuiBox-root MuiBox-root-3 Styled(MuiBox)-root-2"
      classes="[object Object]"
    >
      <div
        class="MuiBox-root MuiBox-root-5 Styled(MuiBox)-root-4"
        classes="[object Object]"
        style="width: 68%;"
      />
      <span
        class="MuiTypography-root MuiTypography-subtitle2"
      >
        Ã˜ 3.4 (3)
      </span>
    </div>
  </div>
</DocumentFragment>
`;
